scripts/translate_static.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Would suggest adding `set -e` to ensure this doesn't fail and continue.

Reply:
The same could be said for `scripts/build.sh`
However i know that fails on `echo "Testing schemas"` in Jenkins as it doesnt have docker.
so maybe `echo "Testing schemas"` doesn't belong as part of `build.sh`?

Reply:
Sorry about the ramblingness of that comment 😄